uniffi/src/bindings/python/gen_python.rs

```diff
@@ -109,6 +117,7 @@ mod filters {
         Ok(match type_ {
             TypeReference::U32 => "4".to_string(),
             TypeReference::Double => "8".to_string(),
+            TypeReference::String => format!("4 + len({})", nm),
```

Comment:
This one definitely gives you the number of unicode characters rather than the number of utf8 bytes, which is definitely wrong.